### PR TITLE
Feature/kgfu02/new group module

### LINKED ISSUE #23: Frontend - Create and style "create new user type" module on dashboard page

### PR DESCRIPTION
**Pull request by kgfu02**
### Administrative Info
[GitHub Issue #]
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
What changes did you make?
- created new group module

### Testing
How did you confirm your changes worked? 
- tested by viewing module in browser using npm start

### Confirmation of Change 
Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.

| Create group module |
| --- |
| Screen Shot 2022-02-06 at 12 31 21 PM |

### CODE REVIEW

**Review by navidboloorian — CHANGES_REQUESTED** [review]
Great stuff as usual Kevin! It's not necessary to add the createGroup component to router. Instead, just have it on the dashboard page because that is where it will be used.

**Comment by kgfu02** [comment]
I'm not sure why but moving the module to the dashboard messed up the styling slightly. @navidboloorian Do you know why this is happening?
Screen Shot 2022-02-10 at 11 43 50 PM

**Comment by navidboloorian** [comment]
> I'm not sure why but moving the module to the dashboard messed up the styling slightly. @navidboloorian Do you know why this is happening? Screen Shot 2022-02-10 at 11 43 50 PM

I think there might be some menu specific styling going on. Check the styles sources with inspect element. There might also be some wild card styling (*{whatever}) that applies to every element. If that's the case will likely need to be tweaked.

**Review by navidboloorian — APPROVED** [review]
LGTM

File: frontend/src/components/CreateGroup.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+import "../css/CreateGroup.css";
+/**
+ * Renders create group module
+ *
+ * @return jsx for create group module
+ */
+function CreateGroup() {
+  return (
+    <div className="modal">
+      <div className="modal-content">
```
Comment (wllmwu) [inline frontend/src/components/CreateGroup.js:21]: I see there's no state or anything in this component yet. Assuming that's supposed to be implemented in a future task

File: frontend/src/index.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { BrowserRouter, Routes, Route } from "react-router-dom";
 import Dashboard from "./pages/Dashboard";
 import Admin from "./pages/Admin";
 import Settings from "./pages/Settings";
+import CreateGroup from "./components/CreateGroup";
```
Comment (wllmwu) [inline frontend/src/index.js:7]: Remove import if not used

**Review by wllmwu — CHANGES_REQUESTED** [review]
Looks good to me, assuming the functionality of this component + integration into dashboard will be implemented in the future. Just remember to remove the import from `index.js` if it's not used there. 👍

File: frontend/src/components/CreateGroup.js (hunk 1 above)
Comment (kgfu02) [reply frontend/src/components/CreateGroup.js:21]: yep, just did the styling for now

**Review by wllmwu — APPROVED** [review]
Looks LGTM to me!

**Pull request merged**